editors/code/src/installation/download_file.ts

```diff
@@ -21,16 +22,16 @@ export async function downloadFile(
     const res = await fetch(url);
 
     if (!res.ok) {
-        console.log("Error", res.status, "while downloading file from", url);
-        console.dir({ body: await res.text(), headers: res.headers }, { depth: 3 });
+        log.error("Error", res.status, "while downloading file from", url);
+        log.error({ body: await res.text(), headers: res.headers }, { depth: 3 });
```

Comment:
Depth won't work with for `console.error()`, `console.dir()` accepts options argument as the second parameter.
```suggestion
        log.error({ body: await res.text(), headers: res.headers });
```
Let's erase it from here (it is only necessary for a terminal (i.e. tty) logging and maybe output channel logging in future (not sure), since Developer Tools enables you to inspect objects by expanding them to an arbitrary level.